service/project.go

```diff
@@ -1009,3 +1015,87 @@ func verifyAliasExists(alias, projectId string, newAliasDefinitions []model.Proj
 	}
 	return false, nil
 }
+
+func bbProjectIsValid(projName string, proj evergreen.BuildBaronSettings) error {
+	webHook, _, _ := model.IsWebhookConfigured(projName, "")
+	webhookConfigured := webHook.Endpoint != ""
+
+	if !webhookConfigured && proj.TicketCreateProject == "" {
+		msg := "ticket_create_project and taskAnnotationSettings.FileTicketWebhook endpoint cannot both be blank"
```

Comment:
These messages shouldn't be logged critical anymore, since the project isn't able to save the invalid configuration (critical logs get sent to evergreen-ops-alerts). We can just return the error.